bin/migrate

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+
+require "bundler/setup"
+require "octokit"
+require "pathname"
+
+require "pt_to_github"
+
+
+if ARGV.length < 2
+  puts "Usage: bin/migrate <path_to_csv> <repo_name>"
+  exit 1
+end
+
+csv_path = Pathname.new(ARGV[0])
+repo_name = ARGV[1]
+
+# Can use either login/password or access_token
+# client = Octokit::Client.new(login: ENV["GITHUB_USERNAME"], password: ENV["GITHUB_PASSWORD"])
+client = Octokit::Client.new(access_token: ENV["GITHUB_ACCESS_TOKEN"])
+
+# A map of name in PT to github usernames.
+# It'll try and assign the issue to the github user
+# if it finds a match to the owners in PT.
+user_map = {
+  "Sean Collings" => "seanrcollings",
```

Comment:
Rather than hard coding users, maybe we can use a yml file?

Reply:
Something like that sounds good to me. I'm not going to have time to add that so you could merge this in and add that in another PR, or push the changes up on this branch :D